n/nextcloud-client/stone.yaml

```diff
@@ -0,0 +1,39 @@
+name: nextcloud-client
+version: "4.0.1"
+release: 2
```

Comment:
Needs to be 1 when we are adding a new recipe.

If this is the newest commit in your tree, you can edit the stone.yaml file to contain release: 1 (instead of 2), run `just build`, git add the changed files, and then run `git commit --amend` and then finally `git push -f`.

You are almost done.